src/views/partials/landing.js

```diff
@@ -1,6 +1,18 @@
 import { getMessage } from '../../utils/fluent.js'
+import chartTestData from '../../../src/client/js/components/circle-chart-test-data.js'
 
 export const landing = data => `
+<div style='display: grid; grid-template-columns: 1fr 1fr 1fr; gap: var(--padding-lg);'>
+  ${chartTestData.map(data => `
+    <circle-chart
+      class='chart-test'
+      data='${JSON.stringify(data.data)}'
+      is-donut='${JSON.stringify(data.isDonut)}'
+      show-percentage-for='${data.showPercentageFor}'
```

Comment:
The `show-percentage-for` attribute doesn't render correctly without `is-donut="true"`.  In the UX discussion, there was only ever a request for 2 states: donut with percent, and pie-chart.  This makes sense to me: If percent is not needed, the graph seems more legible as a pie-chart.

Suggestion: I think we should remove the `isDonut` attribute altogether and include the donut style as part of `showPercentFor`.  This leaves 3 attributes: `title`, `data`, and `show-percentage-for`.

Nit: we might reduce wordiness by using `show-percent-for`.

Reply:
Updated!